config/isr.py

```diff
@@ -105,7 +105,9 @@
 
 config.doNanInterpAfterFlat = False
 
-config.doAmpOffset = True
+# Calculate and apply amp offset corrections.
+config.doMeasureAmpOffset = True
```

Comment:
I think I'm about to comment on the other PR that these options should be `config.ampOffset.doMeasureAmpOffset`

Reply:
I thin you meant `config.ampOffset.doApplyAmpOffset`?

Reply:
Done!